BrickKit/bricks/src/main/java/com/wayfair/brickkit/brick/ViewModel.kt

```diff
@@ -0,0 +1,52 @@
+package com.wayfair.brickkit.brick
+
+import androidx.databinding.BaseObservable
+import com.wayfair.brickkit.brick.DataModel.DataModelUpdateListener
+import java.io.Serializable
+
+/**
+ * The object used to bind information in a [ViewModelBrick].
+ */
+abstract class ViewModel<DM : DataModel> protected constructor(
+    open val dataModel: DM
+) : BaseObservable(), DataModelUpdateListener, Serializable {
```

Comment:
Serializable? 😞

Reply:
gonna tackle that at some point but not yet

Reply:
I guess the ViewModel version of that ticket is gonna be easier than the DataModel version